### PR TITLE
feat(curriculum): add mood board lab for React props

### LINKED ISSUE #667: Add react props lab to main repo with tests
Here is the prototype 
https://github.com/freeCodeCamp/CurriculumExpansion/pull/634

We need to add that to the main repo with the user stories and tests

### PR DESCRIPTION
**Pull request by zairahira**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes https://github.com/freeCodeCamp/CurriculumExpansion/issues/667

### CODE REVIEW

**Review by jdwilkin4 — COMMENTED** [review]
Hey @zairahira !

Thanks for picking up these two react labs.
Oliver's PR to have the react code in the separate file was just merged in.
So we now can have a `.jsx` file just for react instead of having everything in the HTML file.

**Comment by coyotes133413** [comment]
God Bless. Fixing the errors.

**Comment by zairahira** [comment]
@moT01,

The tests ran fine 👍️

File: curriculum/challenges/english/25-front-end-development/lab-mood-board/673b3d6b7ef7318eef926d5a.md [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+# --hints--
+
+You should export a `MoodBoardItem` component.
+
+```js
+const mockedComponent = Enzyme.mount(React.createElement(window.index.MoodBoardItem));
+assert.lengthOf(mockedComponent.find('MoodBoardItem'), 1);
+```
+
+Your `MoodBoardItem` component should return a `div` with a class of `mood-board-item` at it's top-level element.
```
Comment (zairahira) [inline curriculum/challenges/english/25-front-end-development/lab-mood-board/673b3d6b7ef7318eef926d5a.md:49]:
```suggestion
Your `MoodBoardItem` component should return a `div` with a class of `mood-board-item` at its top-level element.
```

File: curriculum/challenges/english/25-front-end-development/lab-mood-board/673b3d6b7ef7318eef926d5a.md [hunk 2]
```diff
@@ -0,0 +1,354 @@
...
+id: 673b3d6b7ef7318eef926d5a
+title: Build a Mood Board
+challengeType: 14
+dashedName: build-a-mood-board
+demoType: onClick
+---
+
+# --description--
+
+A mood board is a collage of images and text. It is used to convey a general idea, goal, or feeling about a particular topic.
```
Comment (Ksound22) [inline curriculum/challenges/english/25-front-end-development/lab-mood-board/673b3d6b7ef7318eef926d5a.md:11]:
```suggestion
A mood board is a collage of images and text that conveys a general idea, goal, or feeling about a particular topic.
```

File: curriculum/challenges/english/25-front-end-development/lab-mood-board/673b3d6b7ef7318eef926d5a.md [hunk 3]
```diff
@@ -0,0 +1,354 @@
...
+
+In this lab, you will create a mood board using reusable React components. The CSS has already been provided for you.
+
+Fulfill the user stories below and get all the tests to pass to complete the lab.
```
Comment (Ksound22) [inline curriculum/challenges/english/25-front-end-development/lab-mood-board/673b3d6b7ef7318eef926d5a.md:15]:
```suggestion
Fulfil the user stories below and get all the tests to pass to complete the lab.
```

**Comment by Ksound22** [comment]
Other than the typo I found, this is good to go in!

**Review by moT01 — APPROVED** [review]

**Review by Ksound22 — APPROVED** [review]
LGTM 🚀

**Pull request merged**